### ISSUE TITLE
Consolidate common servlet code

### ISSUE THREAD

**Issue by pmpowers-usgs** [usgs/nshmp-haz-ws#133]
Every service has repeated NSHMP-specific code in it that could be consolidated in a factory or parent class; NshmpServlet or similar.

**Comment by pmpowers-usgs** [comment]
Create NshmpServlet extends HttpServlet class and overrid:
```
@Override
service(req, resp) {
  // modify response and do other nshmp specific work
  super.service(req, resp);
}
```

**Comment by pmpowers-usgs** [comment]
Resolved in #172
